packages/web-react/src/components/Dropdown/demo/DropdownDefault.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Ref } from 'react';
+import { DropdownRenderProps } from '../../../types';
+import { Button } from '../../Button';
+import { Dropdown } from '..';
+import DropdownContentFactory from './DropdownContentFactory';
+
+const dropdownContent = [
```

Comment:
as this is the same everywhere, couldn't we have it only in one place?

Reply:
good point, moved it to separate file 👍